DesktopEdge/MainWindow.xaml.cs

```diff
@@ -1065,10 +1069,10 @@ public partial class MainWindow : Window {
 			Application.Current.Properties.Add("CurrentTunnelStatus", e.Status);
 			e.Status.Dump(Console.Out);
 			this.Dispatcher.Invoke(() => {
-				if (e.ApiVersion != DataClient.EXPECTED_API_VERSION) {
+				/*if (e.ApiVersion != DataClient.EXPECTED_API_VERSION) {

```

Comment:
why commented out? this on purpose? if so - why leave the dead code?

Reply:
It looks like we are not using the ApiVersion anymore. So I didnt send this value from the ZET. I will remove this block